src/mgr/DaemonServer.h

```diff
@@ -21,6 +21,7 @@
 
 #include "common/Mutex.h"
 #include "common/LogClient.h"
+#include "common/errno.h"
```

Comment:
why would you include this file in the header file, but to use `cpp_strerror()` in `.cc`?